pkg/operator/etcd_assets/bindata.go

```diff
@@ -660,6 +660,34 @@ ${COMPUTED_ENV_VARS}
         name: cert-dir
       - mountPath: /var/lib/etcd/
         name: data-dir
+  - name: rollbackcopier
+    image: ${OPERATOR_IMAGE}
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command:
+      - /bin/sh
+      - -c
+      - |
+        #!/bin/sh
+        set -euo pipefail
+
+        export ETCD_NAME=${NODE_NODE_ENVVAR_NAME_ETCD_NAME}
+        exec cluster-etcd-operator rollbackcopy
+    resources:
+      requests:
+        memory: 60Mi
```

Comment:
just curious how you came up with this value

Reply:
Do you recommend a different number? Wanted to use a lower number than the main etcd container, since it wakes up every 5 seconds and most intense operations are performed every hour. So the requirements are minium.

Reply:
metrics seem to show it at about 14Mi - 20Mi I was not sure if the number was based on what you saw. I think the value is reasonable considering we don't have all usage metrics. IE upgrades.
```
sum(avg_over_time(container_memory_working_set_bytes{container="rollbackcopier",pod!="", namespace=~"openshift-etcd.*"}[5m])) BY (pod, namespace)
```